src/pages/LoginAndRegister/GuestAccountClaim.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<system-page>
+		<div class="page-content">
+			<h1 class="featured-text">
+				One last thing!
+			</h1>
+			<p>
+				To finish creating your account, please enter your first and last name below.
+			</p>
+			<form id="guestAccountClaimForm" action="." @submit.prevent.stop="claimGuestAccount">
+				<label class="input-label" for="firstName">
+					First name
+				</label>
+				<input id="firstName" v-model="firstName" type="text" required @valid="firstName = $event">
+				<label class="input-label" for="lastName">
+					Last name
+				</label>
+				<input id="lastName" v-model="lastName" type="text" required @valid="lastName = $event">
+				<KvButton class="claim-button" type="submit">
+					Done
+				</KvButton>
+			</form>
+		</div>
+	</system-page>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import SystemPage from '@/components/SystemFrame/SystemPage';
+import KvButton from '@/components/Kv/KvButton';
+import completeGuestAccountClaim from '@/graphql/mutation/completeGuestAccountClaim.graphql';
+
+export default {
+	components: {
+		SystemPage,
+		KvButton,
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			firstName: '',
+			lastName: ''
+		};
+	},
+	computed: {
+		formData() {
+			return [
+				`firstName=${encodeURIComponent(this.firstName)}`,
+				`lastName=${encodeURIComponent(this.lastName)}`,
+			].join('&');
+		}
+	},
+	methods: {
+		claimGuestAccount() {
+			this.apollo.mutate({
```

Comment:
We won't have an authentication token at this point, so we won't be able to make this graphql call. What we'll have to do is make a separate page that makes this graphql call and then redirects the user to their final destination, but let's do that in a separate ticket. For this ticket, remove the `this.apollo.mutate` call.